domains/rizwan.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "rizwan3d",
+        "email": "allgame45@hotmail.com"
+    },
+    "records": {
+        "CNAME": "rizwan3d.github.io/rizwan3d"
```

CNAME doesnt allow any trailing slashes (/)
```suggestion
        "CNAME": "rizwan3d.github.io"
```